apis/iam/v1beta1/doc.go

```diff
@@ -0,0 +1,22 @@
+/*
+Copyright 2019 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package v1beta1 contains managed resources, such as
+// ServiceAccount, for IAM services.
+// +kubebuilder:object:generate=true
+// +groupName=iam.gcp.crossplane.io
+// +versionName=v1beta1
+package v1beta1
```

Comment:
> What do you think about introducing new managed resources at v1alpha1, even if we're pretty confident they're already v1beta1 quality? We've been a little inconsistent here in the past, but GKE node pools are an example of a resource we'd consider to be beta quality that were introduced at v1alpha1.

I feel like we are consistent in the sense that no resource became `v1beta1` before a `v1alpha1` phase and this will be part of the [conformance](https://github.com/crossplane/crossplane/pull/1293/files#diff-f89755c114f4759c4b605d9744c77e6eR15) doc.

I agree that we probably won't need to change this resource much but I'm worried about the use cases. Just like AWS, GCP also has [some functions](https://cloud.google.com/iam/docs/reference/rest/v1/projects.serviceAccounts/signBlob) that are independently called. So, I'd be on the side of vetting it at least one release and the best scenario is to use it in a common scenario, integrated with other resources to see how well it stands before v1beta1.

Reply:
I agree with v1alpha1, I honestly think I named it this way due to copy/pasta from the cache operator and forgot to revisit it.